packages/connect-query/src/connect-query-key.ts

```diff
@@ -61,8 +61,5 @@ export const makeConnectQueryKeyGetter =
   (typeName: ServiceType['typeName'], methodInfoName: MethodInfo['name']) =>
   <I extends Message<I>>(
     input?: DisableQuery | PartialMessage<I> | undefined,
-  ): ConnectQueryKey<I> => [
-    typeName,
-    methodInfoName,
-    input === disableQuery || !input ? {} : input,
-  ];
+  ): ConnectQueryKey<I> =>
+    [typeName, methodInfoName, input === disableQuery || !input ? {} : input];
```

Comment:
We should not touch code in this PR, even if it is just formatting. Can you revert?